nsqadmin/static/js/views/app.js

```diff
@@ -133,9 +134,9 @@ var AppView = BaseView.extend({
             var node = new Node({
                 'name': nodeName
             });
-            node.tombstoneTopic(topicName).done(function() {
-                window.location.reload(true);
-            });
+            node.tombstoneTopic(topicName)
+                .done(function() { window.location.reload(true); })
+                .fail(this.handleAJAXError.bind(this));
         });
```

Comment:
Does this actually work?

The `this` in `.fail(this.handleAJAXError.bind(this))` is referring to whatever context `bootbox` uses when it calls this callback. (I'm guessing `this` will be incorrectly set to `window` - the result of calling an instance's method without a context.) You probably need to add `.bind(this)` to the `bootbox` callback, as well.

For example:

```
bootbox.confirm(txt, function(result) {
    ...
}.bind(this));
```